grafana/provisioning/datasources/datasources.yml

```
@@ -0,0 +1,29 @@
+    jsonData:
+      maxLines: 1000
+
+  - name: Zipkin
+    type: zipkin
+    access: proxy
+    url: http://zipkin:9411
+    jsonData:
+      tracesToLogs:
+        datasourceUid: 'Loki'
+        tags: ['service.name', 'trace.id', 'span.id']
+        mappedTags: [{ key: 'service.name', value: 'service' }]
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)